apps/framework-cli/deploy/kubernetes/moose-deployment.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       containers:
       - name: web
-        image: pnxtech/moose-deployment-x86_64-unknown-linux-gnu:latest
+        image: 514labs/moose-deployment-x86_64-unknown-linux-gnu:latest
```

Comment:
This is a specific deployment image for us right? ie that should be a private image

Reply:
yes, will be private by default.